docs/userguide/campaigns.md

```diff
@@ -1,25 +1,156 @@
-* **Numerical tolerance**: This is a flag for forcing numerical measurements to be within a pre-defined tolerance. Note that the setting of the tolerances is controlled as a part of the respective parameter.
-* **Previously done batches and fits** In case that a campaign builds upon previously condiucted experiments, it is possible to provide the number of previously done batches and fits.
+When constructing a campaign, it is necessary to provide two objects:
+A search space (see [class](baybe.searchspace.core.SearchSpace) resp. [user guide](./searchspace))
+and an optimization objective (see [class](baybe.objective.Objective) resp. [user guide](./objective)).
 
-## How to use campaigns
+The search space describes the possible configuration of parameters that can be tested during the campaign.
 
-We have a detailed example explaining how to use the ``Campaign`` object in our examples, so please see [here](./../../examples/Basics/campaign) for all necessary information.
\ No newline at end of file
+```python
+from baybe.parameters import CategoricalParameter, NumericalDiscreteParameter
+from baybe.searchspace import SearchSpace
+
+cat_param = CategoricalParameter(name="cat_param", values=["a", "b", "c"])
+numdisc_param = NumericalDiscreteParameter(name="numdisc_param", values = tuple(range(1,11)))
+
+space = SearchSpace.from_product([cat_param, numdisc_param])
+```
+
+We refer to the [parameters](./parameters) resp. [search spaces](./searchspace) user
+guides for more details.
+
+For the optimization objective, we use a function taking values from 1 to 10 that we aim
+to maximize.
+
+```python
+from baybe.targets import NumericalTarget
+from baybe.objective import Objective
+
+target = NumericalTarget(name="quality", mode="MAX", bounds=(1,10))
+objective = Objective(mode="SINGLE", targets=[target])
+```
+
+Combining the search space and the objective now enables us to create a campaign.
+
+```python
+from baybe.campaign import Campaign
+
+campaign = Campaign(searchspace=space, objective=objective)
+```
+
+### Providing a different strategy
+
+By default, campaigns use the composite
+[`TwoPhaseStrategy`](baybe.strategies.composite.TwoPhaseStrategy).
+Using the optional `strategy` keyword, it is possible to provide a different
+[Strategy](baybe.strategies.base.Strategy).
+For more information on strategies in general, we refer to the corresponding
+[user guide](./strategy).
+
+### Creating a campaign via a configuration
+
+It is also possible to specify a `Campaign` via a configuration string and the function
+[`Campaign.from_config`](baybe.campaign.Campaign.from_config).
+As fully specifying a configuration takes too much, we refer to the corresponding
```

Comment:
"takes too much", strange phrasing, what do you mean? If you eman "too much space" just remove this statement, no need to justify in front of the reader